app/main/posts/views/post-view-data.controller.js

```diff
@@ -55,11 +56,12 @@ function PostViewDataController(
     $scope.selectBulkActions = selectBulkActions;
     $scope.bulkActionsSelected = '';
     $scope.closeBulkActions = closeBulkActions;
-    $scope.selectedPost = {post: null, next: {}};
+    $scope.selectedPost = {post: $scope.$resolve.selectedPost, next: {}};
```

Comment:
Will this error if there is $scope.$resolve.selectedPost is undefined? Is a ternary necessary here?

Reply:
no, because $resolve won't be undefined. An error is raised if you try to access undefinedVar.somethingReal, but not if you try to access somethingReal.undefinedVar
---
But, we probably want a default null on selectedPost tho in the routes, just to  make sure we are not assigning undefined where we want to assign null